include/RAJA/exec-cuda/reduce_cuda.hxx

```diff
@@ -308,58 +333,44 @@ template <size_t BLOCK_SIZE, typename T>
 class ReduceMin< cuda_reduce<BLOCK_SIZE>, T > 
 {
 public:
```

Comment:
Probably my fault, I should do a rebase as this PR is stale and you're seeing weird commit changes. I thought the diffs were smarter (i.e. diff'ed against "current" develop)